### PR TITLE
Remove outdated comment

### LINKED ISSUE #6702: Outdated comment?
This comment mentions a local import due to timing reasons (#4039):

https://github.com/scikit-image/scikit-image/blob/00177e14097237ef20ed3141ed454bc81b308f82/skimage/data/_binary_blobs.py#L51

However, the import was made global in #5872. Is the comment outdated or should the import have remained local?

### PR DESCRIPTION
**Pull request by lagru**
## Description

The imports are not longer there. Closes #6702.

## Checklist

- A descriptive but concise pull request title
- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [Unit tests](https://scikit-image.org/docs/dev/development/contribute.html#testing)
- A gallery example in `./doc/examples` for new features
- [Contribution guide](https://scikit-image.org/docs/dev/development/contribute.html) is followed

## Release note

Summarize the introduced changes in the code block below in one or a few sentences. The
summary will be included in the next release notes automatically:

```release-note
Remove outdated comment.
```

### CODE REVIEW

**Review by jarrodmillman — APPROVED** [review]

File: skimage/data/_binary_blobs.py [hunk 1]
```diff
@@ -49,9 +49,6 @@ def binary_blobs(length=512, blob_size_fraction=0.1, n_dim=2,
     >>> blobs = data.binary_blobs(length=256, volume_fraction=0.3)
 
     """
-    # filters is quite an expensive import since it imports all of scipy.signal
-    # We lazy import here
-
```
Comment (mkcor) [inline skimage/data/_binary_blobs.py:54]: I would keep the empty blank line though.

**Comment by lagru** [comment]
> I've edited your PR description to include release notes, but I wasn't sure:
> 
> If I should have left the period out;

Doesn't matter. [changelist handles both cases.](https://github.com/scientific-python/changelist/blob/75176df34ab72e99768ada505ad36827c4ee569d/src/changelist/_format.py#L129). Also, if the PR description doesn't provide a `release-note` block, the PR title is used as a fall back. But I acknowledge that this may currently be a bit confusing. Once changelist matures I hope for better documentation in that regard.

> if this PR would be considered so minor that it wouldn't be mentioned in the release notes?

My feeling is that mentioning very minor PRs such as in the release notes or changelog is mostly noise. With how changelist and our approach is currently setup, this will be a simple line in some section at the end of the document. One could very easily collapse such sections to "This release contains X maintenance related pull requests."

File: skimage/data/_binary_blobs.py (hunk 1 above)
Comment (lagru) [reply skimage/data/_binary_blobs.py:54]: Huh? I'm -0 but that doesn't feel like the default to me.

**Comment by lagru** [comment]
I went ahead and merged this expecting this to be a very minor uncontroversial change with lazy consensus. :wink:

**Pull request merged**